develop/TOOLS/DIS/Protocols_as_a_Visual_Studio_solution.md

```diff
@@ -42,6 +42,10 @@ A DataMiner protocol Visual Studio solution consists of the following folders:
 
 - *Solution items*: This folder contains the protocol XML. The main difference between a solution protocol XML file and a regular protocol XML file is that the former does not contain the QAction C# code (as this is now in the QAction projects in the Visual Studio solution).
 
+> [!NOTE]
+> As from DIS v2.41, an information bar will allow you to convert existing solutions that make use of the Class Library generation feature.
+> This information bar will appear when a Class Library project (i.e. a project named "QAction_ClassLibrary" or "AutomationScript_ClassLibrary") is detected in a protocol or Automation script solution. As soon as you click *Fix*, the Class Library project will be removed and the references to the project will be replaced by references to the automatically generated Class Library project (with ID 63000).
```

Comment:
```suggestion
> This information bar will appear when a Class Library project (i.e. a project named "QAction_ClassLibrary" or "AutomationScript_ClassLibrary") is detected in a protocol or Automation script solution. As soon as you click *Fix*, the Class Library project will be removed and the references to the project will be replaced by references to the automatically generated Class Library project (default with ID 63000).
```

Reply:
Default it will be with ID 63000, but if at the time of the QAction/Exe generation, that ID was already in use, it will be 63001 (or higher until it found a free ID).